documentation/book/con-resource-kafka.adoc

```diff
@@ -0,0 +1,810 @@
+This object contains the topic operator configuration.
+Without this object, the Cluster Operator does not deploy the topic operator.
+It is still possible to deploy the topic operator by creating appropriate {ProductPlatformName} resources.
+
+The YAML representation of the 'topicOperator` has no mandatory fields and if the value is an empty object
+("{ }"), the Cluster Operator will deploy the topic operator with the default configuration.
+
+The configurable fields are the following :
+
+`image`::
+The Docker image to be used by the Topic Operator.
+The default value is determined by the value specified in the xref:STRIMZI_DEFAULT_TOPIC_OPERATOR_IMAGE[`STRIMZI_DEFAULT_TOPIC_OPERATOR_IMAGE`] environment variable of the Cluster Operator.
+
+`watchedNamespace`::
+The {ProductPlatformName} namespace in which the topic operator watches for topic ConfigMaps. Default is the namespace
+where the topic operator is running.
+
+`reconciliationIntervalMs`::
+The interval between periodic reconciliations in milliseconds. Default is 900000 (15 minutes).
+
+`zookeeperSessionTimeoutMs`::
+The Zookeeper session timeout in milliseconds. Default is 20000 milliseconds (20 seconds).
+
+`topicMetadataMaxAttempts`::
+The number of attempts for getting topics metadata from Kafka. The time between each attempt is defined as an exponential
+back-off. You might want to increase this value when topic creation could take more time due to its larger size (i.e.
+many partitions / replicas). Default is `6`.
+
+`resources`::
+An object configuring the resource limits and requests for the topic operator container. The accepted JSON format is
+described in the xref:resources_json_config-{context}[Resource limits and requests] section.
+
+`affinity`::
+Node and Pod affinity for the Topic Operator, as described in the xref:node-pod-affinity-{context}[Node and Pod affinity].
+The format of the corresponding key is the same as the content supported in the Pod `affinity` in {ProductPlatformName}.
+
+`tlsSidecar.image`::
+The Docker image to be used by the sidecar container which provides TLS support for Topic Operator.
+The default value is determined by the value specified in the xref:STRIMZI_DEFAULT_TLS_SIDECAR_TOPIC_OPERATOR_IMAGE[STRIMZI_DEFAULT_TLS_SIDECAR_TOPIC_OPERATOR_IMAGE`] environment variable of the Cluster Operator.
+
+`tlsSidecar.resources`::
+An object configuring the resource limits and requests for the sidecar container which provides TLS support for the Topic Operator.
+For information about the accepted JSON format, see xref:resources_json_config-{context}[Resource limits and requests].
+
+.Example Topic Operator JSON configuration
+[source,json]
+----
+{ "reconciliationIntervalMs": "900000", "zookeeperSessionTimeoutMs": "20000" }
+----
+
+More information about these configuration parameters in the related xref:topic-operator-{context}[Topic Operator] documentation page.
```

Comment:
Note for self: Divya to add a link to the OpenShift docs if available or revisit if not available.

Reply:
Same as above - we want the link to include the value:

xref:topic-operator-str[Topic Operator]